components/Layout/Images/UserImage.vue

```diff
@@ -48,9 +42,3 @@ export default Vue.extend({
   }
 });
 </script>
-
```

Comment:
Why this was removed?

Reply:
Per [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering):

> Suitable algorithms include nearest-neighbor and other non-smoothing scaling algorithms such as 2×SaI and hqx-family algorithms. This value is intended for pixel-art images, such as in browser games.

None of these algorithms are suited for reducing image sizes properly, and they are targeted more towards pixel art than towards proper images, which user images would generally be.

They would cause pixelization, which is not acceptable, in my opinion.

Reply:
What about swithcing to ``high-quality```?

Reply:
Just a question, why is it needed?

Reply:
@ThibaultNocchi If I recall correctly I added it because Firefox image scalation added more pixelation than Chrome's.

Reply:
Here are FF (left) and Chromium (right) both zoomed in at 200%, so I'm not sure it's still needed :p
![print_screen_2021-02-09-14-57-17](https://user-images.githubusercontent.com/1619359/107373814-3985ed00-6ae7-11eb-8e81-da6b36d7ec3b.png)